BusBoardCSharp/src/BusBoard/BusBoardOperations.cs

```
@@ -0,0 +1,21 @@
+
+namespace BusBoard{
+    class BusBoardOperations{
+
+        public async Task GetBusBoardArrivals(){
+            try{
+                List<ArrivalPredictions> arrivalPredictions = await TflClient.GetLiveArrivalPredictions();
+                DisplayOutput displayOutput = new DisplayOutput();
+                displayOutput.DisplayNextFiveBuses(SortAndFilterList(arrivalPredictions));
+            }
+            catch(Exception exception){
+                Console.WriteLine(exception.Message);
+            }
```

This catch block is redundant. Just let the program crash in this case as we can't give any useful diagnostic info. Remember the flow chart from the logging and error handling presentation. If we can't fix the problem, or add any new information, then just let it crash